src/components/ChoreStep.js

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+import Checkbox from '@mui/material/Checkbox';
+import IconButton from '@mui/material/IconButton';
+import CommentIcon from '@mui/icons-material/Comment';
+
+export default function CheckboxList(props) {
+  const [checked, setChecked] = React.useState([0]);
+  const {sampleSteps} = props
+
+  const handleToggle = (value) => () => {
```

Comment:
It looks like this function, returns a function. I think you're doing that to bind the variable value and then pass that function into the onClick. I think you could also do `handleToggle.bind(null, step)`

I think they are probably equivalent from a performance standpoint. I'm not sure.